microsoftgraph/microsoft-graph-explorer-v4
# fix: remove the replacement of escape characters in json response formatting

## Overview

Removes replacement of escaped characters in the stringified json.
closes #3158

## Testing Instructions

* run the query `https://graph.microsoft.com/v1.0/appCatalogs/teamsApps?$expand=*`
* Check the response preview tab to have well formatted json
